common/enrolment_helper.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime, timedelta
+from logging import getLogger
+from random import randint
+
+from structlog import wrap_logger
+
+from controllers.collection_exercise_controller import create_and_execute_collection_exercise
+from controllers.survey_controller import create_survey, create_classifiers
+
+logger = wrap_logger(getLogger(__name__))
+
+
+def generate_new_iac_code(context):
+    """Generates and returns a unique iac code."""
+
+    test_unique_id = str(randint(100000, 200000))
```

Comment:
Might be worth looking at https://hashids.org/ (implementation https://github.com/davidaurelio/hashids-python) for unique short IDs. Probably ok for now though

Reply:
will do